demonstrations/covalent_cloud_gpu.metadata.json

```
@@ -0,0 +1,41 @@
+{
+    "title": "Running GPU-accelerated quantum circuit simulations on Covalent Cloud using PennyLane",
+    "authors": [
+        {
+            "id": "ara_ghukasyan"
+        },
+        {
+            "id": "santosh_kumar_radha"
+        }
+    ],
+    "dateOfPublication": "2024-03-18T00:00:00+00:00",
+    "dateOfLastModification": "2024-03-18T00:00:00+00:00",
+    "categories": [
+        "Quantum Machine Learning"
+    ],
+    "tags": [],
+    "previewImages": [
+        {
+            "type": "thumbnail",
+            "uri": "/_static/demonstration_assets/covalent_cloud_gpu/thumbnail_covalent_cloud_gpu.png"
+        },
+        {
+            "type": "large_thumbnail",
+            "uri": "/_static/large_demo_thumbnails/thumbnail_large_covalent_cloud_gpu.png"
+        }
+    ],
+    "seoDescription": "Explore the process of running GPU-accelerated quantum circuit simulations on Covalent Cloud using PennyLane.",
+    "doi": "",
+    "canonicalURL": "/qml/demos/covalent_cloud_gpu",
+    "references": [],
+    "basedOnPapers": [],
+    "referencedByPapers": [],
+    "relatedContent": [
+        {
+            "type": "demonstration",
+            "id": "tutorial_univariate_qvr",
+            "weight": 1.0
+        }
+    ],
+    "hardware": []
```

```suggestion
    "hardware": [
        {
            "id": "covalent",
            "link": "https://docs.covalent.xyz/docs/cloud/tutorials-cloud/pennylane_tutorial/",
            "logo": "/_static/hardware_logos/covalent.png"
        }
    ]
```

This suggestion can be applied after https://github.com/PennyLaneAI/qml/pull/1112 is merged in (cc @ikurecic)